src/core/Akka.Remote/Serialization/MessageContainerSerializer.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Akka.Remote.Serialization
 {
-    public class MessageContainerSerializer : Serializer
+    /*TODO: this class is not used*/public class MessageContainerSerializer : Serializer
```

Comment:
This is used via reflection in remote.conf
In every such case, I think we should have some test that actually touches the class via code and do some verification.